git/go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/fluxcd/pkg/git
 go 1.24.0
 
 replace (
-	github.com/fluxcd/pkg/auth => ../auth
```

Comment:
Was this removed by mistake? If we don't replace it here, then we can't debug the git package with local changes to auth.

Reply:
No, it was removed intentionally because the git package depends on the old auth/azure code. I don't want to change the git libraries to use the new auth/azure package now in this PR, I want to do it separately when we implement the RFC for the git APIs down the road, so I'm freezing the packages that depend on the old auth/azure code for now. I'll re-add this line when we do it.

Reply:
Reverted and changed the packages depending on auth/azure to use the new code, thanks!